test/typing/component-function-export.spec.ts

```diff
@@ -8,23 +8,21 @@ export type ConditionalSlotState = {
   mustShowSlot: boolean
 }
 
-export type ConditionalSlotComponent = RiotComponent<
-  ConditionalSlotProps,
-  ConditionalSlotState
->
-
-export default {
-  css: null,
-
-  exports: withTypes<ConditionalSlotComponent>(() => ({
+const exports = () =>
+  (withTypes as WithTypes<ConditionalSlotProps>)({
```

Comment:
this is a breaking change. Is there a way to improve the types without introducing breaking changes?